addons/sale/views/sale_views.xml

```diff
@@ -73,13 +73,50 @@
         </record>
 
         <record id="product_template_form_view" model="ir.ui.view">
-            <field name="name">product.template.product.website.form</field>
+            <field name="name">product.template.form.sales</field>
             <field name="model">product.template</field>
+            <field name="priority">30</field>
             <field name="inherit_id" ref="product.product_template_form_view"/>
             <field name="arch" type="xml">
                 <xpath expr="//page[@name='sales']" position="attributes">
                     <attribute name="invisible">0</attribute>
                 </xpath>
+                <xpath expr="//group[@name='email_template_and_project']" position="before">
+                    <group string="Invoicing" name="invoicing">
+                        <field name="invoice_policy" widget="radio"/>
+                        <field name="service_type" widget="radio" invisible="True"/>
```

Comment:
```suggestion
                        <field name="service_type" invisible="True"/>
```

Reply:
Yes yes, I will remove it, it isn't the priority and I just got to confirm with concerned people that it can be removed ;).

Reply:
Yes, I just saw this will be in v14 (As this PR points to master)

Reply:
This may be retargeted to 13.0, before or post release, depending on our workload and managers decision.

Reply:
It won't be in 13 finally, as indeed we can't change those things post release in stable :confused:.

Reply:
Not even some of them which are more than Improvements?(Like where fields are not visible which are useful)

Reply:
I still have to speak about some of those views with Product Owners, and if a "fix" is decided, that could maybe land in v13.  But the removal of product for code simplification won't land in v13 ^^.